packages/connectivity/src/scp-cf/destination/destination-cache.ts

```diff
@@ -10,7 +10,29 @@ const logger = createLogger({
   messageContext: 'destination-cache'
 });
 
-const DestinationCache = (cache: Cache<Destination>) => ({
+export interface DestinationCacheType {
```

Comment:
General: I think we should make a class out of this. Duplicating type and implementation doesn't make much sense to me. But we can do that later, I guess.